modules/installation-gcp-api-services.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_gcp/installing-gcp-account.adoc
+
+[id="installation-gcp-api-services_{context}"]
+= Enabling API services in GCP
+
+Your Google Cloud Platform (GCP) project requires access to several API services
+to complete {product-title} installation.
+
+.Prerequisites
+
+* You created a project to host your cluster.
```

Comment:
This prereq could probably be:
* A project to host your cluster.

Reply:
This might be a good issue to discuss in a future team meeting - should prereqs be _thing_ or _do a thing_.